src/views/AddItem.jsx

```diff
@@ -29,19 +32,42 @@ export function AddItem({ listToken }) {
 
 	// Store values inside itemData and sent over to database
 	// otherwise log an error if request fails
-	const onFormSubmit = (event) => {
-		event.preventDefault();
+	const onFormSubmit = (e) => {
+		e.preventDefault();
 
 		// if list token is not provided, notify user and exit function
 		if (!listToken) {
 			notifyNoToken();
 			return;
 		}
 
+		// We are looking for an array of items that matches the user input
+		// We use `toLowerCase` and `replaceAll` to account for edge cases,
+		// such as casing and extra spaces
+		const nameMatchesArr = data.some((item) => {
+			return (
+				item.name &&
+				item.name.replaceAll(' ', '') ===
+					itemName.toLowerCase().replaceAll(' ', '')
+			);
+		});
+
+		// The user is notified if the input matches an item in db
+		// Item does not get added; exits the function
+		if (nameMatchesArr) {
+			notifyInList();
+			return;
+		}
+
```

Comment:
Try checking just for space, I am able to add an empty (space) item to db

https://user-images.githubusercontent.com/44907238/217408843-254ee49c-b60d-462f-8475-fea4f8e2dfe7.mov

Reply:
Thanks!  Knew we probably missed something.